Cargo.toml

```diff
@@ -129,6 +129,8 @@ panic = "unwind"
 [profile.production]
 inherits = "release"
 lto = true
+opt-level = 3
```

Comment:
this should be implied by `inherits = "release"` above

Reply:
So you suggest having `opt-level = 3` in the `release` profile ?
If so, then yes, we may inherit from release and do the same for `profile.dev.package.backtrace` ?

Reply:
Oh you mean the **default** for release includes `opt-level = 3`. I don't think it is a bad thing to specify it. It makes it clear and prevent surprises should the default ever change.